### ISSUE TITLE
Show that navigation simulation is currently active

### ISSUE THREAD

**Issue by i315315** [i315315/amos2022ws02-automotive-test-app#14]
## User story
1. As a User
2. I want to be able to see, whether the test app is currently simulating a navigation 
3. So that I know at all times which use case is currently active.

## Acceptance criteria
* It is visible in the app whether the app is currently running a navigation simulation.
* It is visible in the app whether the navigation simulation currently is in the focus of the app (and not running in the background).
* It is possible to see the current status from all screens in the app.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
